src/main/java/org/fsg1/fmms/backend/app/Main.java

```diff
@@ -11,41 +11,61 @@
  * Main class.
  */
 public final class Main {
-    // Base URI the Grizzly HTTP server will listen on
-    static final String BASE_URI = "http://localhost:8080/fmms/";
 
+    /**
+     * Private constructor.
+     * Class should never be instantiated.
+     */
     private Main() {
     }
 
     /**
-     * Starts Grizzly HTTP server exposing JAX-RS endpoints defined in this application.
-     *
+     * Starts Grizzly HTTP server exposing JAX-RS resources defined in this application.
      * @return Grizzly HTTP server.
      */
     public static HttpServer startServer() {
-        // create a resource config that scans for JAX-RS endpoints and providers
-        // in endpoints package
+        Configuration config = Configuration.fromEnv();
+
+        // create a resource config that scans for JAX-RS resources and providers
+        // in resources package
         final ResourceConfig rc = new ResourceConfig();
-        rc.register(new AppBinder());
+
+        AppBinder di = new AppBinder();
+        di.bind(config).to(Configuration.class);
```

Comment:
Can't you put this in the actual AppBinder class?

Reply:
No, it is not the responsibility of the AppBinder to create the app configuration.